Custom AS support in integrator.md (#1593)
oktadeveloper/okta-blog: _source/blog/integrator-setup-adoc.adoc

|`install`
|`false` removes 'Sign up for account' sentence

|`customAuthServer`

> |The name of your custom authorization server (e.g., `my-auth-server`)

Reviewer: ```suggestion
|The name of your custom authorization server (e.g., `default`)
```

Since our default name is... `default`

++++
<div class="separator"></div>
++++

=== Custom Authorization Server

When you need to use a custom authorization server instead of the default one, add the `customAuthServer` parameter:

----
{% raw %}{% include setup/integrator.md type="spa" framework="React" 
   loginRedirectUri="http://localhost:3000/callback" 

>    customAuthServer="my-auth-server" %}{% endraw %}

Reviewer: ```suggestion
   customAuthServer="default" %}{% endraw %}
```

----

This will render the following HTML:

++++
<div class="separator"></div>
++++
{% include setup/integrator.md type="spa" framework="React" 
   loginRedirectUri="http://localhost:3000/callback" 

>    customAuthServer="my-auth-server" %}

Reviewer: ```suggestion
   customAuthServer="default" %}
```
